rviz_mesh_tools_plugins/src/MeshDisplay.cpp

```diff
@@ -111,12 +119,10 @@ MeshDisplay::MeshDisplay()
     RMW_QOS_POLICY_LIVELINESS_SYSTEM_DEFAULT,
     RMW_QOS_LIVELINESS_LEASE_DURATION_DEFAULT,
     false
-  })
-{
-  // mesh topic
-  m_meshTopic = new rviz_common::properties::RosTopicProperty(
-      "Geometry Topic", "", QString::fromStdString(rosidl_generator_traits::name<mesh_msgs::msg::MeshGeometryStamped>()),
-      "Geometry topic to subscribe to.", this, SLOT(updateMeshGeometrySubscription()), this);
+  };
```

Comment:
(optional)
I would also be fine with changing the default QoS to something that has better compatibility (i.e. `volatile` durability; maybe use system default QoS?). Since the QoS settings can be saved with the rviz config, not having `transient_local` durability would not hurt our use-case.

Reply:
I used the system default QoS. It still can be changed in RViz and should be storable via RViz configs.